config/cypress/support/index.js

```diff
@@ -3,3 +3,14 @@ import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command'
 addMatchImageSnapshotCommand({
    customSnapshotsDir: '../../tests/visual/screenshots'
 })
+
+// Command to test Image Screenshots and match/check for visual regressions
+// Screenshot testing only takes place in headless environment where
```

Comment:
I think I know what you mean, but this comment is a little confusing since the consistency part is more about it running on  the same host  OS (Mac) than it is about running without the browser, right?  Ie. wouldn't the `runner` env run  on  the same computer also produce valid screenshots?